localization/ekf_localizer/include/ekf_localizer/hyper_parameters.hpp

```diff
@@ -24,36 +24,41 @@ class HyperParameters
 {
 public:
   explicit HyperParameters(rclcpp::Node * node)
-  : show_debug_info(node->declare_parameter("show_debug_info", false)),
-    ekf_rate(node->declare_parameter("predict_frequency", 50.0)),
+  : show_debug_info(node->declare_parameter<bool>("node.show_debug_info")),
+    ekf_rate(node->declare_parameter<double>("node.predict_frequency")),
     ekf_dt(1.0 / std::max(ekf_rate, 0.1)),
-    tf_rate_(node->declare_parameter("tf_rate", 10.0)),
-    publish_tf_(node->declare_parameter("publish_tf", true)),
-    enable_yaw_bias_estimation(node->declare_parameter("enable_yaw_bias_estimation", true)),
-    extend_state_step(node->declare_parameter("extend_state_step", 50)),
+    tf_rate_(node->declare_parameter<double>("node.tf_rate")),
+    publish_tf_(node->declare_parameter<bool>("node.publish_tf")),
+    enable_yaw_bias_estimation(node->declare_parameter<bool>("node.enable_yaw_bias_estimation")),
+    extend_state_step(node->declare_parameter<int>("node.extend_state_step")),
     pose_frame_id(node->declare_parameter("pose_frame_id", std::string("map"))),
```

Comment:
Please delete the default value of "pose_frame_id" as well, and change it to be specified in param.yaml.
In addition, remove the "pose_frame_id" from the [`ekf_localizer.launch.xml`](https://github.com/autowarefoundation/autoware.universe/blob/bd4b5ca2fc48c972fabb5fff68bdcbf5b1aeece8/localization/ekf_localizer/launch/ekf_localizer.launch.xml#L28) :pray:

Reply:
@KYabuuchi Thank you for your review. I modified it just now. Please check it.

Reply:
@Motsu-san  Thanks. But you need to modify `misc.sub_schema.json` too :pray:

Reply:
@KYabuuchi Yeah, I'm aware of it. Before this pull request, there was not "pose_frame_id" in the Misc table. That's why I didn't. I would add "pose_frame_id" description to README if allowed. So, I would like to know what it means and how to set if needed. Could you?
![image](https://github.com/autowarefoundation/autoware.universe/assets/83898149/3ece5da6-eb58-4d3b-94ec-ced76452c4f6)

Reply:
@Motsu-san  Please add it to the Misc table. "pose_frame_id" means `Parent frame_id of EKF output pose`.

Reply:
@KYabuuchi OK. I'll do that. I understood that it is needed to be changed when just using other frame_id of output pose than "map". Thanks.